crypto/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "crypto"
+version = "0.1.0"
+description="Miden Cryptographic primitives"
+authors = ["miden contributors"]
+readme="README.md"
+license = "MIT"
+repository = "https://github.com/0xPolygonMiden/crypto"
+categories = ["cryptography", "no-std"]
+keywords = ["miden", "crypto", "hash", "merkle"]
+edition = "2021"
+
+[dependencies]
+winterfell = { git = "https://github.com/novifinancial/winterfell"}
+winter_utils = { version = "0.4", package = "winter-utils" }
+rand_utils = { version = "0.4", package = "winter-rand-utils" }
```

Comment:
Should this go into `dev-dependencies` as well?

Also, minor nit: there is an extra blank line here.